src/node/glossary.ts

```diff
@@ -40,11 +42,17 @@ export interface SetupServerApi {
    */
   resetHandlers(...nextHandlers: RequestHandler[]): void
 
+  /**
+ * Returns lists of all active request handlers.
+ * @see {@link https://mswjs.io/docs/api/setup-server/list-handlers `server.list-handlers()`}
+ */
+  listHandlers(): RequestHandler<RequestHandlerDefaultInfo, MockedRequest<DefaultBodyType>, any, MockedRequest<DefaultBodyType>>[]
```

Comment:
If we go to the readonly list route (which I think we should), let's adjust this return type as well.